### ISSUE TITLE
Change Line Spacing in Voting Process Descriptions

### ISSUE THREAD

**Issue by ferranrego** [vocdoni/ui-scaffold#460]
The current line spacing in the descriptions of the processes is excessively large. I believe it would enhance readability and design consistency if we reduce the line spacing. This adjustment would make the text more compact and easier to follow, aligning better with our overall design aesthetics.

![image](https://github.com/vocdoni/ui-scaffold/assets/3515290/91608a04-47e3-4296-a963-772370fe29f9)

**Comment by ferranrego** [comment]
Just to confirm, this issue hasn't been resolved yet, right?

**Comment by elboletaire** [comment]
Merged today morning

**Comment by ferranrego** [comment]
Great, I just checked it!

Just a small observation before we close this issue. Currently, the _line height_ and the _paragraph space_ are the same, and this difficult a bit the readability. If I'm correct, the line-height is at 150% of the font-size. Then we can potentially set the paragraph spacing to around 175% so it improves the readability and the overall look. What do you think?

**Comment by elboletaire** [comment]
> Great, I just checked it!
> 
> Just a small observation before we close this issue. Currently, the _line height_ and the _paragraph space_ are the same, and this difficult a bit the readability. If I'm correct, the line-height is at 150% of the font-size. Then we can potentially set the paragraph spacing to around 175% so it improves the readability and the overall look. What do you think?

Yes, although that's a different issue technically speaking, but let's reuse this one.

@gerouvi I'm assigning it to you, if you are blocked at some moment with the markdown issue, try to do this in between.

**Comment by jpaulet** [comment]
Already solved.
